terraform/modules/aws/modules/infra/main.tf

```diff
@@ -0,0 +1,620 @@
+provider "aws" {
+  region = var.region
+}
+
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+locals {
+  cos-cluster-name    = "cos-cluster"
+  cos-cloud-name      = "cos-cloud"
+  cos-controller-name = "cos-controller"
+}
```

Comment:
Would it make a better reference if those were vars, and we committed a `.tfvars` file as well?